features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/MetadataMergeScenario.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MetadataMergeScenario.swift
+//  iOSTestApp
+//
+//  Created by Jamie Lynch on 28/05/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import Foundation
+
+class MetadataMergeScenario: Scenario {
+
+    override func startBugsnag() {
+        self.config.autoTrackSessions = false;
+        super.startBugsnag()
+    }
+
+    override func run() {
+        let error = NSError(domain: "MetadataMergeScenario", code: 100, userInfo: nil)
+        Bugsnag.notifyError(error) { (event) -> Bool in
+            // set placeholder values
+            event.addMetadata("initialValue", key: "nonNullValue", section: "custom")
```

Comment:
I think that configuring the initial values on `BugsnagClient` and just overriding in the callback would make for a more powerful e2e test.

Reply:
Updated to configure initial values on `Bugsnag` instead.